src/utils.ts

```diff
@@ -0,0 +1,5 @@
+export function sleep(ms: number): any {
```

Comment:
everything is using arrow function here so lets keep the code standard consistent
```const sleep = (ms: number) => (callback: Function) => setTimeout(callback, ms)```